src/hl7v2/schema.clj

```diff
@@ -160,7 +205,12 @@
 
 (comment
 
-  (gen-structure "ORU_R01")
+  (require '[clojure.pprint :as pp])
+
+  (spit
+   "test/hl7v2/data/ORU_R01.edn"
```

Comment:
Why ORU_R01 will be an edn's file?

Reply:
here I am spitting the oru hiccup as an edn file for testing purpose, but later, we will open another repo for models, each trigger event will have it's own edn.